subcommittees/mentoring/proposal-final-exercise-of-instructor-training.md

```diff
@@ -0,0 +1,41 @@
+# Checkout Session for Instructor Training
+
+**This is a rough draft.**
+
+## Terminology
+
+-   Instructor Trainer: someone appointed by the Software Carpentry
+    Steering Committee to lead a Instructor Training event.
+-   Trainee: anyone that attended a Software Carpentry Instructor Training event.
+-   [Pre-Checkout discussion session](#pre-checkout-discussion-session):
+    meeting between a Instructor Trainer and some trainees
+    to discuss one of Software or Data Carpentry lessons.
+-   [Checkout session](#checkout-session): meeting between a Instructor Trainer
+    and a trainee to conclude Instructor Training.
+
+## Pre-Checkout Discussion Session
+
+1.  Sign up for one pre-checkout discussion session
+    that will cover the topic he/she want to cover at the checkout session.
+
+2.  Join the pre-checkout discussion session.
+
+## Checkout Session
+
+**Before attend a checkout session all trainees are required
+to join a pre-checkout discussion session.**
+
+1.  Sign up for one checkout session.
+
+2.  Join the checkout session.
+
+3.  Deliver a 5min lesson about one of the lessons on the curriculum.
+
+    The trainee decide which lesson he/she want to be checkout,
+    but the Instructor Trainer leading the checkout session
+    decides which segment of that lesson the trainee has to teach.
+
+## After Checkout Session
+
+The Instructor Trainer leading the checkout session decides
+whether the trainee has passed, or should re-try at a later date.
```

Comment:
Maybe with an additional sentence like ”And if you pass, TODO will assign your Instructor badge”?  Or is the way these sessions fit into the broader training / badging framework out of scope for this file?